app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field form-group col-md-6 my-2">
+    <%= f.label :email, class: 'form-label' %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email", class: 'form-control' %>
+  </div>
+
+  <div class="field form-group col-md-6 my-2">
+    <%= f.label :password, class: 'form-label' %><br />
+    <%= f.password_field :password, autocomplete: "current-password", class: 'form-control' %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field form-group my-2" >
+      <%= f.check_box :remember_me, class: 'form-check-input' %>
+      <%= f.label :remember_me, class: 'form-check-label' %>
+    </div>
+  <% end %>
+
+  <div class="actions my-2">
+    <%= f.submit "Log in", class:'btn btn-success' %>
+  </div>
+<% end %>
+
+<%= render "devise/shared/links" %>
```

Comment:
- [ ]  @Team Great work adding the login page, however the design slightly differs from the design given, the buttons should be the same width as the filed form, check the comparison below:


Given Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://github.com/katarighe/blog-app/assets/61408860/e99fbec7-d31c-41b2-bd32-e6e01f9284bd) |  ![image](https://github.com/Nelofarzabi/blog_app/assets/61408860/6453f19d-96aa-43d7-a480-88fb677efcdd)